demo/src/main/webapp/VAADIN/themes/demo/addons.scss

```diff
@@ -1,17 +1,7 @@
-/* This file is automatically managed and will be overwritten from time to time. */
-/* Do not manually edit this file. */
-
-/* Provided by stepper-2.4.0.jar */
-@import "../../..//VAADIN/addons/stepper/stepper.scss";
-
-
-/* Provided by stepper-2.4.0.jar */
-@import "../../../VAADIN/addons/stepper/stepper.scss";
-
-
-/* Import and include this mixin into your project theme to include the addon themes */
-@mixin addons {
-	@include stepper;
-	@include stepper;
-}
-
+/* This file is automatically managed and will be overwritten from time to time. */
+/* Do not manually edit this file. */
+
+/* Import and include this mixin into your project theme to include the addon themes */
+@mixin addons {

```

Comment:
Does the demo still work without addon.scss import?

Reply:
I guess? It was removed by Gradle, I didn't touch it myself. Reverted anyway.